packages/clipper/src/domtoimage.ts

```diff
@@ -16,41 +16,31 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
-import { cloneNode, isSVGElement } from "./clone.js";
 import { createImage, FetchOptions } from "./fetch.js";
 import { resolveAll } from "./fontfaces.js";
 import { inlineAllImages } from "./images.js";
 import { Options } from "./types.js";
-import { canvasToBlob, delay, escapeXhtml, height, width } from "./utils.js";
-import { cacheStylesheets, inlineStylesheets } from "./styles.js";
+import {
+  canvasToBlob,
+  delay,
+  escapeXhtml,
+  height,
+  width,
+  isSVGElement
+} from "./utils.js";
+import { inlineStylesheets } from "./styles.js";
 
 // Default impl options
 const defaultOptions: Options = {
   inlineOptions: {}
 };
 
 async function getInlinedNode(node: HTMLElement, options: Options) {
-  const { fonts, images, stylesheets, inlineImages } =
-    options.inlineOptions || {};
+  const { fonts, stylesheets, inlineImages } = options.inlineOptions || {};
```

Comment:
We can't ignore `images` since it is used to configure whether we want images in the clip or not. I suggst removing the `img` tags from the cloned node if `images` option is false.